beacon_node/beacon_chain/src/migrate.rs

```diff
@@ -190,6 +198,10 @@ impl<E: EthSpec, Hot: ItemStore<E>, Cold: ItemStore<E>> BackgroundMigrator<E, Ho
         Ok(())
     }
 
+    pub fn process_manual_finalization(&self, notif: ManualFinalizationNotification) {
+        let _ = self.send_background_notification(Notification::ManualFinalization(notif));
```

Comment:
I don't think we ever run store migrator in the foreground other than testing, but probably should support it given we do it  for all 3 other notifications, and if we want to test this with `BeaconChainHarness`, it won't be immediately obvious why this doesn't get run.

Reply:
Good point. We should do this for the clean version of this PR